### PR TITLE
Changed the user base. Issue No. #6320

### LINKED ISSUE #6320: README.md: Update users count badge to 140K
The current badge is showing 80K. Should be updated to 140K

### PR DESCRIPTION
**Pull request by ovshake**
Fixes #6320

### CODE REVIEW

**Comment by wkurniawan07** [comment]
@ovshake, do refer to our [development process](https://github.com/TEAMMATES/teammates/blob/master/docs/process.md). In particular, your PR title and branch name are both wrong, and you should start from a new branch instead of committing on `master` when working on any fix.
Also, you have committed unrelated file changes such as those in `.settings` folder. Please revert those.

**Comment by ovshake** [comment]
@wkurniawan07 
I had some doubts regarding the contributing procedure
1) There are some changes in some of my particular files (e.g. in queue.xml I had to remove certain queues due to mine being an unbilled account in gcloud) Is it okay to ignore them using this command
`git rm --cache <FILENAME>`?
2) In my above PR, I should have created a new branch in my fork (with the appropriate naming) and then made a PR to the master of the main repo. Am I correct? I want to confirm so as to avoid any more errors. :)
Sorry for the late attention as it took me a while to set up the dev environment.

**Comment by wkurniawan07** [comment]
This issue has been resolved, so I have to close this PR.

@ovshake to answer your questions:
1. Is it too much trouble to enable billing for your gcloud account? If you set the spending limit to 0, it is as good as not spending anything but still getting all the goodness of billed accounts.
2. Yes, and I see that you have followed it correctly in your first timer PR, good job!

**Comment by damithc** [comment]
> Is it too much trouble to enable billing for your gcloud account? If you set the spending limit to 0, it is as good as not spending anything but still getting all the goodness of billed accounts.

@wkurniawan07 This requires a credit card. Some of our contributors might not have one. Any way around it?

**Comment by wkurniawan07** [comment]
> Any way around it?

I'm afraid not. The only way is to reduce the number of tasks in the `queue.xml` to 10 or less. That would mean some task queues will not work for these devs, but as long as they work on smaller features this should not have too large of an effect.

**Comment by damithc** [comment]
@wkurniawan07 

> The only way is to reduce the number of tasks in the queue.xml to 10 or less.

Can we survive with fewer task queues? I remember this increase of task queues is due to a recent refactoring. Is the increase strictly necessary?

**Comment by wkurniawan07** [comment]
> Can we survive with fewer task queues? I remember this increase of task queues is due to a recent refactoring. Is the increase strictly necessary?

I can look at ways so that we can survive now, but they come at non-trivial trade-offs:
- We can move some of the queued tasks back to ordinary tasks (i.e. executed as part of `*Action`), essentially moving backwards, with an obvious performance trade-off.
- We can combine some different task queues into one (for example combining `AdminPrepareEmailAddressModeWorkerAction` and `AdminPrepareEmailGroupModeWorkerAction`, which was actually done before), at the expense of code readability and maintainability (e.g why would this class do two entirely different actions based on the parameter passed in?)

Also, as we scale up, it will only be a matter of time before task queue becomes a necessity rather than  premature optimization, so we can only expect the number of tasks to go up as time passes by. Already we have another PR adding one more item to the task queue: #6529.

In short: We can for now, but not possible in the long run.

**Comment by damithc** [comment]
@wkurniawan07 OK, I guess no choice. It's just not nice when developers cannot use their own staging server without a credit card or editing the config file (we need to document how to do that).
